Lesson1/Exercise3/Program.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+
+
+namespace Exercise3
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Введите первую строку:");
+            String first = Console.ReadLine();
+            Console.WriteLine("Введите вторую строку:");
+            String second = Console.ReadLine();
+            Console.WriteLine("Введите третью строку:");
+            String third = Console.ReadLine();
+
+            String firstLowerCase = first.ToLower();
+            String secondLowerCase = second.ToLower();
+            String thirdLowerCase = third.ToLower();
+
+            //Method for startWith First to Second
+            bool CompareFirstSecond()
```

Comment:
You've implemented method inside method (very bad readability) and such method doing only one action that could be performed by system methods
For what ?